chef.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency "mixlib-log", "~> 1.3"
   s.add_dependency "mixlib-authentication", "~> 1.3"
   s.add_dependency "mixlib-shellout", ">= 2.0.0.rc.0", "< 3.0"
-  s.add_dependency "ohai", "~> 8.0"
+  # s.add_dependency "ohai", "~> 8.0"
```

Comment:
Yes, and it would have to be pinned to a minimum version of ohai which includes these changes (probably 8.5?)